### PR TITLE
Add test for NewsResourceCard

### PR DESCRIPTION
**Pull request by calren**

### CODE REVIEW

File: core-ui/src/main/java/com/google/samples/apps/nowinandroid/core/ui/NewsResourceCard.kt [hunk 1]
```diff
@@ -72,6 +74,8 @@ import kotlinx.datetime.toJavaInstant
  * [NewsResource] card used on the following screens: For You, Episodes, Saved
  */
 
+const val NewsResourceCardTestTag = "NewsResourceCardTestTag"
```
Comment (simonpojok) [inline core-ui/src/main/java/com/google/samples/apps/nowinandroid/core/ui/NewsResourceCard.kt:77]: Let's remove test word for the same, just for separation of concern

**Review by simonpojok — APPROVED** [review]

File: core-ui/src/androidTest/java/com/google/samples/apps/nowinandroid/core/ui/NewsResourceCardTest.kt [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        composeTestRule.setContent {
+            NewsResourceCardExpanded(
+                newsResource = newsResource,
+                isBookmarked = false,
+                onToggleBookmark = {},
+                onClick = { }
+            )
+        }
+
+        composeTestRule.onNodeWithTag(NewsResourceCardTestTag).assertHasClickAction()
```
Comment (alexvanyo) [inline core-ui/src/androidTest/java/com/google/samples/apps/nowinandroid/core/ui/NewsResourceCardTest.kt:78]: Can we use the `clickActionLabel` with `onNodeWithContentDescription` as opposed to a test tag? That'd avoiding changing the production code for tests.

**Comment by dturner** [comment]
@calren Please could you address the issue mentioned by @alexvanyo and resolve the merge conflicts.

**Comment by mmoczkowski** [comment]
Closing due to inactivity. If you still want to submit your change create a new PR. Thanks!

**Pull request closed without merging**